packages/application-config/src/substitute-env-variable-placeholders.ts

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
This file is the "same" as the one we currently have in the `mc-html-template` package

Reply:
So we should keep them in sync, remove one or merge them from a utils package?

Reply:
I'll migrate things in a follow up PR. The one in the mc-html-template package will then go away